core/txdb/internal/storage/snapshot.proto

```diff
@@ -10,14 +10,23 @@ message Snapshot {
   repeated StateTreeNode nodes = 1;
 
   // Issuances contains the record of recent issuances for ensuring uniqueness
-  // of issuances.
-  repeated Issuance issuances = 2;
+  // of issuances. DEPRECATED in favor of nonces (below).
+  repeated Issuance issuances = 2 [deprecated=true];
```

Comment:
Done. I knew that renaming a field would not affect the wire format, but I was less sure about renaming a message type. A little research makes me reasonably assured that's safe too.